app/routes/events/components/EventDetail/index.js

```diff
@@ -295,26 +278,66 @@ export default class EventDetail extends Component<Props> {
               </Flex>
             </ContentMain>
             <ContentSidebar>
-              <InfoList items={infoItems} />
+              {event.company && (
+                <div>
+                  <Icon
+                    name="briefcase-outline"
+                    style={{ marginRight: '10px' }}
+                  />
+                  <strong>
+                    <Link to={`/companies/${event.company.id}`}>
+                      {event.company.name}
+                    </Link>
+                  </strong>
+                </div>
+              )}
```

Comment:
I'll just continue the discussion here since its impossible in the other discussion.

I just feel like companies want to be liked to linked to in an accessible place. Do i need to mail bedkom about this? 😬

Reply:
I totally agree that we need a link to the company, but I don't think it needs to be at the very top, above the date and location, which I find much more important.

Reply:
I think that bedkom should have a say in this as it is related to the exposure that they sell to businesses. Sure, the name is exposed in other ways through the page in graphics and the text, but that link is the only place where people can click for more information and find active job llistings. So I think it is fair for it to stay exposed, and if we change it we should consult with other people that might have a say.

Reply:
Yeah, and as of now it is on the top. And it's only there when a company is involved, so maybe of that reason it makes sense to let it be there?

Reply:
I'll send a mail to bedkom, then I can ask about "Arrangør"/"Forfatter" also.